bazel/BUILD.hiredis

```diff
@@ -0,0 +1,47 @@
+COPTS = [] + select({
+    "@bazel_tools//src/conditions:windows": [
+        "-D_CRT_DECLARE_NONSTDC_NAMES=0",  # don't define off_t, to avoid conflicts
+    ],
+    "//conditions:default": [
+    ],
+})
+
+cc_library(
+    name = "_hiredis",
+    hdrs = [
+        "dict.c",
+    ],
+    copts = COPTS,
+)
+
+cc_library(
+    name = "hiredis",
+    srcs = glob(
+        [
+            "*.c",
+            "*.h",
+        ],
+        exclude =
+        [
+            "test.c",
+        ],
+    ),
+    hdrs = glob([
+        "*.h",
+        "adapters/*.h",
```

Comment:
`**/*.h` wildcards?

Reply:
That would pull in the `examples/` directory which we don't want.

Reply:
(Unless you mean I should use `exclude`?)